spiped/spiped.1

```diff
@@ -153,5 +153,13 @@ Print version number.
 .TP
 .B \-1
 Exit after a connection closes.  Used for internal testing.
+.SH SIGNALS
+.B SIGTERM
```

Comment:
@gperciva Go ahead please, I don't want to spend time refining this pull request at the moment. I agree that my current formatting is pretty ugly, though. I tried to find man pages on my system that have a `SIGNALS` section, but could not find any.